app/models/vpago/payment_decorator.rb

```diff
@@ -7,6 +7,22 @@ def self.prepended(base)
       base.after_update :cancel_pre_auth, if: :state_changed_to_failed?
     end
 
+    # @override
+    # order all payment.process! when calling next.
+    def process!
+      return complete! if payment_method.auto_capture? && payment_receive_manually?
+
+      super
+    end
+
+    def payment_receive_manually?
+      check_payment_method? && order.ticket_seller_user?
+    end
+
+    def check_payment_method?
+      payment_method.is_a?(Spree::PaymentMethod::Check)
+    end
+
```

Comment:
All ticket_seller logics are here inside spree_vpago. So we don't have to do it in commissioner yet.